.github/workflows/expire.yml

```diff
@@ -86,6 +86,14 @@ jobs:
                   pull_number: pull['number'],
                   state: 'closed',
                 });
+
+                // Comment on closed PRs.
+                await github.rest.issues.createComment({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: pull['number'],
+                  body: `**\`Access on Demand\`** - request closed due to inactivity for ${{ inputs.expiry_hours }} hours`,
```

Comment:
NIT: `request closed due to inactivity for at least ${{ inputs.expiry_hours }} hours`

Reply:
Thanks, fixed!